service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequest.java

```diff
@@ -120,6 +120,7 @@
 
     private String backurl;
 
+    private int noShipping;
```

Comment:
This is PayPal specific and not common for all payment methods, right? Maybe put it rather into WalletAuthorizationRequest and WalletPreauthorizationRequest classes instead.

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/413/commits/6f1625eeeb82416971e4847793c27599ec452c7e